fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/RelationTransformer.java

```diff
@@ -536,6 +537,9 @@ public LogicalPlan visitTable(TableRelation node, ExpressionMapping context) {
                     new LogicalJDBCScanOperator(node.getTable(), colRefToColumnMetaMapBuilder.build(),
                             columnMetaToColRefMap, Operator.DEFAULT_LIMIT,
                             null, null);
+        } else if (Table.TableType.TEMPORARY_EXTERNAL_TABLE.equals(node.getTable().getType())) {
```

Comment:
1. It's not the temporary external table. Temporary tables have meaning in the database. It exists for sessions.
2. It's better to add more commit message for so large table.